application/src/main/java/uk/nhs/hee/tis/revalidation/service/DoctorsForDBService.java

```diff
@@ -170,7 +170,7 @@ private TraineeInfoDto convert(final DoctorsForDB doctorsForDB) {
         .underNotice(doctorsForDB.getUnderNotice().name()).sanction(doctorsForDB.getSanction())
         .doctorStatus(doctorsForDB.getDoctorStatus().name()) //TODO update with legacy statuses
         .lastUpdatedDate(doctorsForDB.getLastUpdatedDate()).admin(doctorsForDB.getAdmin())
-        .connectionStatus(getConnectionStatus(doctorsForDB.getDesignatedBodyCode()));
+        .tisConnectionStatus(getConnectionStatus(doctorsForDB.getDesignatedBodyCode()));
```

Comment:
`getConnectionStatus` method checks if `DoctorsForDB.designatedBodyCode` exists. If it doesn't exist, set `connectionStatus` to `false`, else set it to `true`. So it looks like the `TraineeInfoDto.connectionStatus` is not from TCS.

Reply:
I think that this line is some kind of copy/paste of old connections logic 😬 this field isn't relevant to recommendations so we should not be calculating it or using it at all here